ML importer test failure - datafeed frequency

While running [TestImportJobs](https://github.com/elastic/beats/blob/7ce70f68f7c6579d931180927ee6a7d2d1b6d510/libbeat/ml-importer/importer_integration_test.go#L117) in libbeat the test is failing.

> "Datafeed frequency [1m] must be a multiple of the aggregation interval [3600000ms]"

The full output from the failed test case is:

```
command [go test -tags integration -cover -coverprofile /tmp/gotestcover-412122985 github.com/elastic/beats/libbeat/ml-importer]: exit status 1
--- FAIL: TestImportJobs (0.73s)
	Error Trace:	importer_integration_test.go:117
	Error:		Received unexpected error "load datafeed under /_xpack/ml/datafeeds/datafeed-test-ml-config. Response body: {\"error\":{\"root_cause\":[{\"type\":\"status_exception\",\"reason\":\"Datafeed frequency [1m] must be a multiple of the aggregation interval [3600000ms]\"}],\"type\":\"status_exception\",\"reason\":\"Datafeed frequency [1m] must be a multiple of the aggregation interval [3600000ms]\"},\"status\":400}: couldn't load json. Error: 400 Bad Request: {\"error\":{\"root_cause\":[{\"type\":\"status_exception\",\"reason\":\"Datafeed frequency [1m] must be a multiple of the aggregation interval [3600000ms]\"}],\"type\":\"status_exception\",\"reason\":\"Datafeed frequency [1m] must be a multiple of the aggregation interval [3600000ms]\"},\"status\":400}"
		
	Error Trace:	importer_integration_test.go:164
	Error:		Should be true
		
	Error Trace:	importer_integration_test.go:173
	Error:		Should be true
		
FAIL
coverage: 63.3% of statements
FAIL	github.com/elastic/beats/libbeat/ml-importer	0.740s
```

Travis log: https://travis-ci.org/elastic/beats/jobs/315410604#L2422

--- a/libbeat/ml-importer/importer_integration_test.go
+++ b/libbeat/ml-importer/importer_integration_test.go
@@ -43,8 +43,6 @@ const sampleJob = `
 const sampleDatafeed = `
 {
     "job_id": "PLACEHOLDER",
-    "query_delay": "60s",
-    "frequency": "60s",
     "indexes": [
       "filebeat-*"
     ],

--- a/libbeat/ml-importer/importer_integration_test.go
+++ b/libbeat/ml-importer/importer_integration_test.go
@@ -165,7 +165,7 @@ func TestImportJobs(t *testing.T) {
 		if datafeed.DatafeedId == "datafeed-test-ml-config" {
 			found = true
 			assert.Equal(t, datafeed.JobId, "test-ml-config")
-			assert.Equal(t, datafeed.QueryDelay, "60s")
+			assert.Equal(t, datafeed.QueryDelay, "87034ms")
 		}
 	}
 	assert.True(t, found)

